src/pilot/deploy-overcloud.py

```diff
@@ -30,18 +36,534 @@
 from identify_nodes import main as identify_nodes
 from update_ssh_config import main as update_ssh_config
 
+logging.basicConfig()
+logger = logging.getLogger(os.path.splitext(os.path.basename(sys.argv[0]))[0])
+
 home_dir = os.path.expanduser('~')
 
 BAREMETAL_FLAVOR = "baremetal"
 
+# Dell nfv configuration global values
+undercloudrc_file_name = "stackrc"
```

Comment:
I think the above variable may no longer be used.  Can you check and remove if this is the case?